packages/Button/__tests__/Button.ssr.spec.jsx

```diff
@@ -1,16 +0,0 @@
-/**
- * @jest-environment node
- */
-
-import React from 'react'
-import { render } from 'enzyme'
-
-import Button from '../Button'
-
-describe('Button server side rendering', () => {
-  it('uses mobile styles first', () => {
-    const button = render(<Button>Submit</Button>)
-
-    expect(button.hasClass('fullWidth')).toBe(true)
-  })
-})
```

Comment:
Did all the SSR specs get deleted?

If so, there is a quick cleanup that can be done. These tests were using the `@jest-environment node` syntax to change the environment from jsdom to node for these tests. 

With these tests deleted, I believe this means we can make these changes:
* Delete the "config/jest/setupEnzyme.js" file as this is no longer needed
* Remove the reference to "setupEnzyme.js" from "config/jest.config.js"

Reply:
The `Box` component still has an SSR test.

Reply:
Why? It probably doesn't need it anymore? If its all based on media queries, then there is no need for the SSR test.

Reply:
Ah, I see, the SSR script just runs a redundant test. I'll remove it then. I also noticed DisplayHeading has one too, so I'll get rid of that as well and clean these up.

Reply:
Hang on, the tooltip still has an SSR test since it still uses the responsive component. Wouldn't this mean we havet to keep the SSR config stuff in?

Reply:
Ahhhhh, ok gotcha. Yep makes sense then. 👍